integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/mysql/TestMysqlTableOperations.java

```diff
@@ -77,9 +80,10 @@ public void testOperationTable() {
             .withNullable(false)
             .build());
     Map<String, String> properties = new HashMap<>();
-    // TODO #804 Properties will be unified in the future.
-    //    properties.put("ENGINE", "InnoDB");
-    //    properties.put(AUTO_INCREMENT, "10");
+    properties.put(ENGINE_KEY, "InnoDB");
+    properties.put(AUTO_INCREMENT, "10");
+    properties.put(DEFAULT_CHARSET_KEY, "utf8mb3");
+    properties.put(COLLATE_KEY, "utf8mb3_general_ci");
```

Comment:
Did you also need to test the default properties

Reply:
But the test image version we used is fixed. I suggest the scenario that needs testing is whether these properties are indeed automatically created by the underlying system.